sdk/identity/identity/review/identity.api.md

```diff
@@ -207,7 +207,7 @@ export const logger: AzureLogger;
 export class ManagedIdentityCredential implements TokenCredential {
     constructor(clientId: string, options?: TokenCredentialOptions);
     constructor(options?: TokenCredentialOptions);
-    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
```

Comment:
This shouldnt have happened...

Reply:
That is so weird ... let me take a look.

Reply:
I created a fresh clone and rebased on latest master and the issue went away after a rebuild. I am still not sure what caused this change in the first place.